messages/add.json

```diff
@@ -0,0 +1,7 @@
+{
+    "commandDescription": "Add custom rules to use while scanning. Rules should have been compiled and tested separately. Users can refer to PMD’s documentation on information and examples to write your own custom rules: https://pmd.github.io/latest/pmd_userdocs_extending_writing_pmd_rules.html",
+    "languageFlagDescription": "Programming language for which custom rules are added. Required flag.",
```

Comment:
I think we should standardize how these JSONs are structured. Take a look at mine for reference. Also, you don't need to specify that the flag is required, because the output of the help command will do that for you.

Reply:
Will do 👍